src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPanelMappingTest.java

```diff
@@ -94,11 +92,46 @@ void testGetAuthorisationSubTypes() {
     @DisplayName("When overrideFields are null getPanelPreferences returns an empty list")
     @Test
     void testGetPanelPreferencesOverrideFieldsNull() {
-        List<PanelPreference> result = HearingsPanelMapping.getPanelPreferences();
+        List<PanelPreference> result = HearingsPanelMapping.getPanelPreferences(caseData, referenceDataServiceHolder);
 
         assertThat(result).isEmpty();
     }
 
+    @DisplayName("When adjournment is enabled, adjournment is in progress, "
+        + " all the 3 panel members are provided "
+        + " and panel member requirement type is provided" 
+        + " then return all the panel member with the requirement type.")
+    @ParameterizedTest
+    @MethodSource("getPanelMemberExcludeStates")
+    void testGetPanelPreferencesOverrideFieldsNotNull(AdjournCasePanelMembersExcluded membersExcluded, RequirementType requirementType) {
+        given(referenceDataServiceHolder.isAdjournmentFlagEnabled()).willReturn(true);
+
+        caseData.setAdjournment(Adjournment.builder().adjournmentInProgress(YesNo.YES)
+            .panelMembersExcluded(membersExcluded)
+            .panelMember1(JudicialUserBase.builder()
+                .idamId("1")
+                .personalCode("TOM")
+                .build())
+            .panelMember2(JudicialUserBase.builder()
+                .idamId("2").personalCode("JERRY")
+                .build())
+            .panelMember3(JudicialUserBase.builder()
+                .idamId("3").personalCode("Tyke")
+                .build())
+            .build());
+        List<PanelPreference> result = HearingsPanelMapping.getPanelPreferences(caseData, referenceDataServiceHolder);
+        assertThat(result).isNotNull();
+        assertThat(result.get(0).getRequirementType()).isEqualTo(requirementType);
+    }
+
+    private static Stream<Arguments> getPanelMemberExcludeStates() {
```

Comment:
Please change to CsvSource

Reply:
Removed the parameterized testing added separate test case for each case.